### ISSUE TITLE
Allow bundle IDs to not begin with "com."

### ISSUE THREAD

**Issue by martinfinke** [olilarkin/wdl-ol#40]
_IPlug_include_in_plug_hdr.h:50_:

``` cpp
#define BUNDLE_ID "com." BUNDLE_MFR "." API_EXT "." BUNDLE_NAME
```

The "com" should be replaced by a BUNDLE_DOMAIN (or similar). Not everyone has a .com domain.

**Comment by olilarkin** [comment]
don't think this is that important TBH. i use com.olilarkin, even though my website is .co.uk

**Comment by olilarkin** [comment]
hmm, maybe

**Comment by martinfinke** [comment]
It is:

> The string should also be in reverse-DNS format. For example, if your company’s domain is Ajax.com and you create an app named Hello, you could assign the string com.Ajax.Hello as your app’s bundle identifier.

[link](https://developer.apple.com/library/mac/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102070)

**Comment by olilarkin** [comment]
addressed in 6143061ed7d39f6b32a229d686b62327358e029b
